original_sdk/tests/e2e/test_async_client_e2e/test_async_allocate_claim_flow.py

```diff
@@ -0,0 +1,52 @@
+import time
+
+import globals as gbl
+
+from original_sdk import OriginalAsyncClient
+from original_sdk.utils import get_random_string
+
+
+class TestClientE2E:
```

Comment:
nit: might want to rename the ones in the async folder to TestAsyncClientE2E, or the name of the test file might be better TestAsyncAllocateClaimFlow